docs/source/gettingStarted.rst

```diff
@@ -164,7 +164,7 @@ SimpleITK has a built in function,
 which can be used for viewing images in an interactive session.
 By default this Show function searches for an installed
 `Fiji <https://fiji.sc>`__ to display images.  If Fiji is not found,
-then it searches for `ImageJ <http://rsbweb.nih.gov/ij/>`__. Fiji/ImageJ were
+then it searches for `ImageJ <http://imagej.nih.gov/ij/>`__. Fiji/ImageJ were
```

Comment:
spelling error? How was this detected?

Reply:
The long, not particularly interesting story, is that it didn't like 'rsbweb', and I didn't really want to put it into the dictionary, so I checked the web address, and imagej works and makes more sense.